ctapipe_io_lst/__init__.py

```diff
@@ -484,26 +492,32 @@ def fill_trigger_info(self, array_event, zfits_event):
         tib_available = lst.evt.extdevices_presence & 1
         ucts_available = lst.evt.extdevices_presence & 2
 
-        # Since at least June 2020, ucts should be more reliable
-        # before there were runs when ucts was missing or was shifted
-        # by one event after an event with ucts_trigger_type == 42.
```

Comment:
The one event jumps occur (not very frequently) also after June 25th, but are fixable